meta-rz-bsp/recipes-core/packagegroups/packagegroup-rz-tools-gpu.bb

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024, Renesas Electronics Corp.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "Graphics packages packagegroup."
+DESCRIPTION = "Set of packages useful for graphics and hw acceleration"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+inherit packagegroup
+
+RDEPENDS:${PN} = " \
+	glmark2 \
+	gtk+3-demo \
+	kmscube \
+	startup \
```

Comment:
Thanks for your contribution!

Directly listing `startup` in this recipe means it cannot be built when the optional meta-rz-panfrost layer is not included in the build, as the `startup` recipe is provided in that layer.

I realised that the renesas-image-demo recipe was already broken in our master branch, so we just fixed that: https://github.com/renesas-rz/rz-community-bsp/commit/c2d7f45c1f22fb8cffdf832af36f1a481f602def

Please could you remove `startup` from the dependencies here and add it to a new file `meta-rz-bsp/dynamic-layers/meta-rz-panfrost/recipes-core/packagegroups/packagegroup-rz-tools-gpu.bbappend` so that it is only included when the meta-rz-panfrost layer is enabled? You can also delete `meta-rz-bsp/dynamic-layers/meta-rz-panfrost/recipes-core/images/renesas-image-demo.bbappend` now that the dynamic dependency is handled in the packagegroup.

Let me know if you have any questions on that.